src/App.js

```diff
@@ -1,25 +1,21 @@
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+import Greeting from "./components/Greeting";
```

Comment:
- [NOT REALLY OPTIONAL] Everything is good except for this. The app crashes at this importation. The file name starts with lowercase `g` but in the importation, we have uppercase `G`. Consider refactoring the file name to start with an uppercase `G`.